clarify name of an index order (#1575)
DataDog/terraform-provider-datadog: docs/resources/logs_index_order.md

# The Datadog Terraform Provider does not support the creation and deletion of index orders. There must be at most one `datadog_logs_index_order` resource
terraform import <datadog_logs_index_order.name> <name>
```

> `<name>` here can be whatever you specify in your code; datadog does not store a name at their server.

Reviewer: ```suggestion
`<name>` can be whatever you specify in your code; Datadog does not store a name on the server.
```
